extras/travis_build.sh

```diff
@@ -280,13 +281,8 @@ echo "MSGBLD0280: Make install completed."
     echo "+               R U N   N E S T   T E S T S U I T E                           +"
     echo "+ + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + +"
     echo "MSGBLD0290: Running make installcheck."
-    if [ "$TRAVIS_PYTHON_VERSION" = "2.7.13" ]; then
-        export PYTHONPATH=$HOME/.cache/csa.install/lib/python2.7/site-packages:$PYTHONPATH
-        export LD_LIBRARY_PATH=$HOME/.cache/csa.install/lib:$LD_LIBRARY_PATH
-    elif [ "$TRAVIS_PYTHON_VERSION" = "3.6.7" ]; then
-        export PYTHONPATH=/usr/lib/x86_64-linux-gnu/:$PYTHONPATH
-        export LD_LIBRARY_PATH=$HOME/.cache/csa.install/lib:$LD_LIBRARY_PATH
-    fi
+    export PYTHONPATH=/usr/lib/x86_64-linux-gnu/:$PYTHONPATH
+    export LD_LIBRARY_PATH=$HOME/.cache/csa.install/lib:$LD_LIBRARY_PATH
```

Comment:
Wouldn't these exports be better placed where the Travis build matrix switches enclose Python and CSA?

Reply:
Thanks for the suggestion, fixed!